src/explore-education-statistics-admin/src/pages/release/edit-release/content/actions.ts

```diff
@@ -0,0 +1,90 @@
+import { EditableContentBlock } from '@admin/services/publicationService';
+import {
+  AbstractRelease,
+  ContentSection,
+} from '@common/services/publicationService';
+import { State } from './ReleaseContext';
+
+type ClearState = { type: 'CLEAR_STATE' };
+type SetState = { type: 'SET_STATE'; payload: State };
+type SetAvailableDatablocks = {
+  type: 'SET_AVAILABLE_DATABLOCKS';
+  payload: Pick<State, 'availableDataBlocks'>;
+};
+export type RemoveBlockFromSection = {
+  type: 'REMOVE_BLOCK_FROM_SECTION';
+  payload: {
+    meta: {
```

Comment:
Refactor out a type for this. Seems like there should be some kind of type that encapsulates the meta and can be reused in other parts of this file. Importantly, we want to name it appropriately so that we're not using a weird `Omit<RemoveBlockFromSection['payload']['meta'], 'blockId'>` type.

Reply:
we now have BlockMeta and SectionMeta types :)